vdb-bench-assembly/plugins/vdb-bench-dataservice/content/css/styles.less

```diff
@@ -706,3 +706,16 @@ button[id^='dropdownKebabRight_'] {
     position: relative;
     top: -4em;
 }
+
+.dsb-wizard-container .modal-header {
+    padding: 0px 15px;
+}
+
+.dsb-wizard-container .wizard-pf-steps-indicator {
```

Comment:
Are the properties added to this not included in standard wizard-pf-steps-indicator class? Might need to add !important after them to ensure they have priority?